smoke-tests/spec/external_dns_spec.rb

```diff
@@ -3,14 +3,14 @@
 # This test can only be ran against live-1. Test clusters do not have enough privileges.
 describe "external DNS", "live-1": true do
   let(:domain) { "integrationtest.service.justice.gov.uk" } # That zone already exists
-  zone = "Z02429076QQMAO8KXV68" # integrationtest.service.justice.gov.uk zone_id
+  zone_id = "Z02429076QQMAO8KXV68" # integrationtest.service.justice.gov.uk zone_id
```

Comment:
This should probably be defined as a constant, to clarify that it's a magic value.